cvat-canvas/src/typescript/editHandler.ts

```diff
@@ -380,6 +368,21 @@ export class EditHandlerImpl implements EditHandler {
     }
 
     private closeEditing(): void {
+        if (this.isEditing && this.editData.state.shapeType === 'polyline') {
+            const { offset } = this.geometry;
+            const head = this.editedShape.attr('points').split(' ').slice(0, this.editData.pointID).join(' ');
+            const stringifiedPoints = `${head} ${this.editLine.node.getAttribute('points').slice(0, -2)}`;
+            const points = pointsToNumberArray(stringifiedPoints)
+                .slice(0, -2)
+                .map((coord: number): number => coord - offset);
+            if (points.length >= 2 * 2) { // minimumPoints * 2
+                const { state } = this.editData;
+                this.onEditDone(state, points);
+            }
+        } else {
+            this.onEditDone(null, null);
+        }
+        this.isEditing = false;
```

Comment:
Probably `release` shall be called inside the condition. Otherwise it leads to the issue, I described in the pull request with polygons

Reply:
Release method needs to be called whenever a polyline is edited, so i think it should be outside. 

As for the issue, I fixed it by cheking the shapeType of the editHandler and pass `{enabled: false}` only if the shapetype is polyline in canvasview.ts. It shall work fine now.